WordPress/Classes/Services/PostService+MarkAsFailedAndDraftIfNeeded.swift

```diff
@@ -9,10 +9,6 @@
     ///     eventually be made private.
     ///
     func markAsFailedAndDraftIfNeeded(post: AbstractPost) {
-        guard post.remoteStatus != .failed, !post.hasRemote() else {
```

Comment:
I think we can keep the first part

```suggestion
    guard post.remoteStatus != .failed else {
```

Reply:
Added back.